### ISSUE TITLE
Ability to select API version per server

### ISSUE THREAD

**Issue by spkane** [ptsochantaris/trailer#416]
If you have Github and Github Enterprise it might be desirable to use API v3 with one server and API v4 with the other server, depending on what version of GHE you have deployed. It would be ideal if you could select the API version per server.

**Comment by ptsochantaris** [comment]
You're absolutely right about this, and it's something I really wanted to do originally, but there was so much to tweak that I ended up going this route to simplify and make the change more manageable :D I can't promise you this will be added super-soon, but it's definitely something I'm hoping to do eventually.
